script.js

```diff
@@ -0,0 +1,59 @@
+alert(`Ну что Погнали :)`);
+
+const nam = prompt("Введите ваше имя:");
+alert(`Привет, ${nam} !`);
+
+const this_year = 2021;

```

Comment:
такие константы обычно пишут UPPER_CASE
```suggestion
const THIS_YEAR = 2021;

```